body.tex

```diff
@@ -107,8 +107,16 @@ \subsection{Registering}
 \end{itemize}
 \validation
 \begin{itemize}
-    \item $\field{O}{id}$ must be unique, \ie not currently in use,
-        between $1$ and $32$ bytes long, and valid \textsf{UTF-8}.
+    \item $\field{O}{id}$ must be unique, \ie not currently in use as an org
+        identifier {\em or} user identifier, and comply with the following
+        rules:
+        \begin{itemize}
+            \item Must be between $1$ and $32$ bytes long.
```

Comment:
What's the encoding, is it ASCII? How about `Must be between $1$ and $32$ ASCII-encoded characters.`? It's the same in 2 places below.

Reply:
It's UTF-8 encoding.

Reply:
The UTF-8 part was removed :( For this range it's the same as ASCII, so it doesn't matter which one do we pick.

Reply:
> The UTF-8 part was removed :(

Ah, indeed, good catch! @cloudhead was that intentional?

Reply:
Yeah, giving free range over full UTF-8 is opening ourselves up to all sorts of issues for our users, eg. non-breaking spaces, look-alike characters etc. Since we're using this to quickly identify one user/org over another, it needs to be unambiguous and clear, and names shouldn't be similar to each other.

Of course, the names are still valid UTF-8, but now it's a much more limited range, within the ASCII charset.

Reply:
Done.